### PR TITLE
Dns scan, region checking refactored to suit different value sets across resources

### PR DESCRIPTION
**Pull request by VaclavK**

### CODE REVIEW

File: src/EShopWorld.Tools/Commands/AzScan/AzScanAllCommand.cs [hunk 1]
```diff
@@ -22,9 +22,10 @@ public override async Task<int> OnExecuteAsync(CommandLineApplication app, ICons
             await Task.WhenAll(
                 GetCompositeCommand<AzScanSqlCommand>().OnExecuteAsync(app, console),
                 GetCompositeCommand<AzCosmosDbScanCommand>().OnExecuteAsync(app, console),
-                GetCompositeCommand<AzRedisScanCommand>().OnExecuteAsync(app, console),
+                GetCompositeCommand<AzScanRedisCommand>().OnExecuteAsync(app, console),
```
Comment (dguerin) [inline src/EShopWorld.Tools/Commands/AzScan/AzScanAllCommand.cs]: I actually prefer the other way you had it. Az [Resouce] Activity. It's more intuitive and follows the same convention as using the az CLI eg:

```
az vm create -n MyLinuxVM -g MyResourceGroup --image UbuntuLTS
```

In any case the servicebus and cosmsodb packages commands are not consistent with the reset that are in the page.

Comment (dguerin) [reply src/EShopWorld.Tools/Commands/AzScan/AzScanAllCommand.cs]: Additionally when the help is lined up nicely, you should be able to get an determine what functionality is in the app for each resource based on just the supported azure resource.

Comment (VaclavK) [reply src/EShopWorld.Tools/Commands/AzScan/AzScanAllCommand.cs]: so as discussed we have the order of "verbs" dictacted by the CLI aggregated unrelated islands of commands together so the higher level verbs differentiates between these functional islands (i.e. scan first before resource type)

Comment (dguerin) [reply src/EShopWorld.Tools/Commands/AzScan/AzScanAllCommand.cs]: yeah makes sense. Once ServiceBus and Cosmosdb names are consistent with the other classes this is all good for me.

**Pull request merged**